src/NServiceBus.Metrics/RawData/OccurrenceWriter.cs

```diff
@@ -0,0 +1,42 @@
+namespace NServiceBus.Metrics.RawData
+{
+    using System;
+    using System.IO;
+
+    static class OccurrenceWriter
+    {
+        const long Version = 1;
+
+        // WIRE FORMAT, Version: 1
+
+        //0                   1                   2                   3
+        //0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
```

Comment:
Previously we shipped all metrics in one message. So this means we'll go from 1 message every n seconds to potentially multiple messages. Can we cope with that additional load? Or would it make more sense to batch them up into one message?